docs/use-cases/azure.md

```diff
@@ -0,0 +1,113 @@
+
+### Cluster Creation
+
+!!! info
+    When working with the Azure CLI, it can help to set a default `location`, `group`, and `acr`.
+    See `az configure --help`, `az configure --list-defaults`, and `az configure --defaults key=value`
+
+The following creates an AKS cluster with some minimal configuration that will work well with Flux:
+
+```sh
+az aks create \
+ --network-plugin="azure" \
+ --network-policy="calico" \
+ --enable-managed-identity \
+ --enable-pod-identity \
+ --name="my-cluster"
+```
+
+## Flux Installation with Azure DevOps Repos
+
+Ensure you can login to [https://dev.azure.com](dev.azure.com) for your proper organization, and create a new repo to hold your
+flux install and other necessary config.
+
+There is no bootstrap provider currently for Azure DevOps Repos,
+but you can clone your Azure Repo, then use the [Generic Git Server](../guides/installation.md#generic-git-server)
+guide to manually bootstrap Flux. (It must be a Git repo; TFVC Repos are not supported by source-controller)
+Take note of the Azure DevOps specific section within the guide.
+
+If you use the generated SSH deploy key from `flux create source git`, ensure it is an RSA key (not an elliptic curve).
+Make sure you're using the `libgit2` provider, compatible with Git Protocol v2, for any `GitRepository` object
+fetching from or updating Azure Repos.
+
+Whether you're using the generated SSH deploy key or a Personal Access Token, the credentials used by
+Flux will need to be owned by an Azure DevOps User with access to the repo.
+Consider creating a machine-user and granting it granular permissions to access what's needed.
+This allows changing user access without affecting Flux.
+Since PAT's expire on Azure DevOps, using a machine-user's login password to authenticate with HTTPS and `libgit2`
+can be a good option that avoids the need to renew the credential while also having the benefit of more granular permissions.
+
+## Helm Repositories on Azure Container Registry
+
+The Flux `HelmRepository` object currently supports [Chart Repositories](https://helm.sh/docs/topics/chart_repository/)
+as well as fetching `HelmCharts` from paths in `GitRepository` sources.
+
+Azure Container Registry has a sub-command ([`az acr helm`](https://docs.microsoft.com/en-us/cli/azure/acr/helm)) for working with
+ACR-Hosted Chart Repositories, but it is deprecated.
+If you are using these deprecated Azure Chart Repositories, you can use Flux `HelmRepository` objects with them.
+
+[Newer ACR Helm documentation](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-helm-repos) suggests
+using ACR as an experimental [Helm OCI Registry](https://helm.sh/docs/topics/registries/).
+This will not work with Flux, because using Charts from OCI Registries is not yet supported.
+
+## Secrets Management with SOPS and Azure Key Vault
```

Comment:
This section could have more clear steps for how to achieve secrets management with `aad-pod-identity` and Azure Key Vault. If these steps were laid out more granularly, could help with understanding

Reply:
These commands are in the SOPS section:
> See the [Mozilla SOPS Azure Guide](../guides/azure.md) for further detail.